content/induction/blocks/cyf-folder/index.md

```diff
@@ -0,0 +1,43 @@
++++
+title = '🧰 Create a CYF folder'
+headless = true
+time = 20
+facilitation = false
+vocabulary=[""]
+emoji= '🧩'
+[objectives]
+1='Navigate and manipulate the file system using a terminal'
+2='Create a cyf directory to store your work over the course'
++++
+
+{{<tabs name="create-cyf-folder">}}
+
+{{% tab name="Create a CYF folder" %}}
+
+You'll need to create a CYF folder to store your projects on the course.
+
+1. Open a terminal on your computer.
+
+For each of the steps below, you'll need to use the command line in your terminal.
+
+Use this [cli documentation](https://www.techrepublic.com/article/16-terminal-commands-every-user-should-know/) to remember terminal commands.
+
+2. In your terminal, print your current working directory.
+
+3. List the files and folders in your current working directory.
+
+You'll need a place to store your work for the CYF course.
+
+4. **M**a**k**e a new **dir**ectory called `cyf` in your home directory.
+
+5. **C**hange **d**irectory into the `cyf` directory.
+
+6. Double check you're in the right place by **p**rinting your current **w**orking **d**irectory.
+
+{{% /tab %}}
+
+{{<tab name="📼 Watch: How to create a CYF folder">}}
+{{% youtube %}}https://www.youtube.com/watch?v=JFX8UfXWmok{{% /youtube %}}
```

Comment:
This video isn't being rendered in the UI - same is true for this tab in production too. https://curriculum.codeyourfuture.io/fundamentals/prep/